test/debugger/helper-debugger-repl.js

```diff
@@ -34,11 +34,11 @@ let quit;
 
 function startDebugger(scriptToDebug) {
   scriptToDebug = process.env.NODE_DEBUGGER_TEST_SCRIPT ||
-                  common.fixturesDir + '/' + scriptToDebug;
+                  `${common.fixturesDir}/${scriptToDebug}`;
```

Comment:
I am in doubt if there is a minimum consensus for this. If there are 2-3 +1 and no strong -1, I can try to refactor with `path.join()`.